app/src/include/firebase/app.h

```diff
@@ -55,11 +55,6 @@ namespace internal {
 class AppInternal;
 }  // namespace internal
 
-#ifdef _STLPORT_VERSION
```

Comment:
Do we want a similar warning for gnustl at this point? Or would that be potentially too noisy?

Reply:
I think we should have the warning. We've had deprecation warnings sit there for months before.

Reply:
Done.